src/lwc/geBatchWizard/geBatchWizard.js

```diff
@@ -342,20 +342,39 @@ export default class geBatchWizard extends NavigationMixin(LightningElement) {
     */
     handleSave() {
         this.isLoading = true;
-        const dataImportBatchObjectInfo = this.dataImportBatchCreateDefaults.data.objectInfos[
-            this.dataImportBatchName
-        ];
-        const recordDefaults = this.dataImportBatchCreateDefaults.data.record;
-        let recordObject = generateRecordInputForCreate(recordDefaults, dataImportBatchObjectInfo);
-        recordObject = this.setFieldValues(recordObject);
-
         if (this.recordId) {
-            this.handleRecordUpdate(recordObject);
+            this.handleRecordUpdate(this.buildUpdateRecord());
         } else {
-            this.handleRecordCreate(recordObject);
+            this.handleRecordCreate(this.buildCreateRecord());
         }
     }
 
+    /**
+     * @description Builds a new data import batch record to be saved
+     * @returns {DataImportBatch record}
+     */
+    buildCreateRecord() {
```

Comment:
Are these changes related to the overwriting of batch defaults?  Is there a separate section in the test charter to test how these changes effect batch default values?

Reply:
Yes, these changes are related to batch field values getting reset through the batch wizard. Yes, it is called out in the test charter.